json_schemas/buyer-email-domains.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "additionalProperties": false,
+  "properties": {
+    "domainName": {
+      "pattern": "^(?!:\/\/)([a-zA-Z0-9-_]+\\.)*[a-zA-Z0-9][a-zA-Z0-9-_]+\\.[a-zA-Z]{2,64}?$",
```

Comment:
While I'm pretty sure that actual government buyer domains will be ascii and digits, I'm not sure we should be so restrictive here - internationalised domain names are a real thing, and I don't see a good reason to exclude them here.

I think the main things we should check are that there is no `@` and at least one `.` - beyond that all bets are off. (We could have a length restriction - probably sensible... 255?)

Reply:
https://blog.farrant.me/getting-an-emoji-domain-in-2015/ 😹 

Happy to change this. I think I'd just nabbed the regex above from an example online. I think the TLD part is ok (apparently the max length for TLDs is 64 otherwise the DNS can't resolve!), so how about something like 

```
"^([^@\s])+\.[a-zA-Z]{2,64}$"
```

This should skip `@` and whitespace, and require at least one dot before a sane TLD. My regex powers are pretty dire so any advice gratefully received!